libs/newsletter/guides/Overview.md

```diff
@@ -0,0 +1,86 @@
+# Daffodil Newsletter
+
+The Daffodil Newsletter library manages your newsletter subscription service. It utilizes multiple different ecommerce API drivers to make connecting your application's newsletter simple while also providing tools to help manage your UI.
+<!--talk about supported platforms--> 
+
+## Table of Contents
+- [Getting Started](#getting-started)
+  - [Setting up your AppModule](#setting-up-your-appmodule)
+  - [Utilizing inside your Component](#utilizing-inside-your-component)
+- [Sending a Newsletter Subscription](#sending-a-newsletter-subscription)
+  - [Using the facade](#using-the-facade)
+- [Live Demo](#live-demo)
+
+
+
+# Getting Started
+This overview assumes that you have already set up an Angular project and have gone through the [Newsletter installation guide](). If you have not, we recommend you do that first.
+
+## Setting up your AppModule
+
+To get started, import the `StoreModule` and the `DaffNewsletterModule` at the top of your app.module file.
+```typescript
+import { DaffNewsletterModule } from '@daffodil/newsletter';
+import { StoreModule } from '@ngrx/store';
+```
+Then import the `DaffNewsletterModule` in your app.module. Afterwards, also import `StoreModule.forRoot({})`, as this will be relevant later on when utilizing the redux and state management features of the newsletter module.
+
+```typescript
+@ngModule({
+  imports:[
+    DaffNewsletterModule,
+    StoreModule.forRoot({})
+  ]
+})
+
+```
+
+## Utilizing inside your component
+
+The Daffodil Newsletter Module creates a `DaffNewsletterFacade` to wrap all the complexities of the library into one place. This facade will handle sending your newsletter subscription to your application's backend and can also be configured to help assist your UI.
```

Comment:
> and can also be configured to help assist your UI.

and can also be utilized to build your UI with behaviors common to a newsletter.